config/oidc.php

```diff
@@ -21,10 +21,54 @@
     'client_secret' => env('OIDC_CLIENT_SECRET', ''),
 
     /**
-     * Only needed when response of user info endpoint is encrypted.
-     * This is the path to the JWE decryption key.
+     * Configuration for client authentication.
      *
-     * You could add multiple decryption key paths comma separated.
+     * By default, the client authentication method used is either `client_secret_basic`, `client_secret_post`,
```

Comment:
Great documentation, a nice addition would be to mention the [Client Authentication section RFC](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication).